Documentation/disaster-recovery.md

```diff
@@ -75,20 +172,31 @@ kubectl -n rook-ceph edit configmap rook-ceph-mon-endpoints
 
 In the `data` element you will see three mons such as the following (or more depending on your `moncount`):
 ```
-data: rook-ceph-mon-a=10.100.35.200:6789;rook-ceph-mon-b=10.100.35.233:6789;rook-ceph-mon-c=10.100.35.12:6789
+data: a=10.100.35.200:6789;b=10.100.13.242:6789;c=10.100.35.12:6789
 ```
 
 Delete the bad mons from the list, for example to end up with a single good mon:
 ```
-data: rook-ceph-mon-b=10.100.35.233:6789
+data: b=10.100.13.242:6789
 ```
 
 Save the file and exit.
 
+Now we need to adapt a Secret which is used for the mons and other components.
+The following `kubectl patch` command is an easy way to do that. In the end it patches the `rook-ceph-config` secret and updates the two key/value pairs `mon_host` and `mon_initial_members`.
+
+```bash
+mon_host=$(kubectl -n rook-ceph get svc rook-ceph-mon-b -o jsonpath='{.spec.clusterIP}')
+kubectl -n rook-ceph edit secret rook-ceph-config -p '{"data": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "b"}}'
```

Comment:
can we use variables here such as `${good_mon_id}`? The `b` might be overlooked otherwise.